packages/breadboard-web/src/boards/simplest.ts

```diff
@@ -4,42 +4,44 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { Schema, base, board } from "@google-labs/breadboard";
-import { core } from "@google-labs/core-kit";
+import {
+  annotate,
+  board,
+  input,
+  object,
+  output,
+} from "@breadboard-ai/build";
+import { invoke } from "@google-labs/core-kit";
 
-const inputSchema = {
-  type: "object",
-  properties: {
-    text: {
-      type: "string",
-      title: "Prompt",
-      description: "The prompt to generate a completion for",
-      examples: ["Tell me a fun story about playing with breadboards"],
-    },
-  },
-  required: ["text"],
-} satisfies Schema;
+const text = input({
+  type: "string",
+  title: "Prompt",
+  description: "The prompt to generate a completion for",
+  examples: ["Tell me a fun story about playing with breadboards"],
+});
+
+const generator = input({
+  $id: "math-question",
+  title: "Generator",
+  type: annotate(object({}), {
+    behavior: ["board"],
+  }),
+  description: "The URL of the generator to call",
+  default: { kind: "board", path: "gemini-generator.json" },
+});
 
-const outputSchema = {
-  type: "object",
-  properties: {
-    text: {
-      type: "string",
-      title: "Response",
-      description: "The completion generated by the LLM",
-    },
-  },
-  required: ["text"],
-} satisfies Schema;
+const invoker = invoke({ $id: "gemini", $board: generator, instructions: text }).unsafeOutput("text");
```

Comment:
Suggest calling this variable `response` instead of `invoker`. (It's not really an *invoker*, which sounds like something that invokes something; rather it's a value -- the result of calling invoke. Plus it's actual title is "Response" later on.)